modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerTest.java

```diff
@@ -111,498 +98,17 @@ public void tearDown() throws Exception {
     }
 
     @Test
-    public void testCreateZoneWithAutoAdjust() throws Exception {
```

Comment:
Why did you remove this and similar tests?

Reply:
There are two major points here:
1. First of all, DZM has it's own API that should be tested.
2. Highly likely some zone specific validation logic will be presented in DZM itself, so it should be tested though corresponding API.

Reply:
Discussed personally - delete them.